requirements-cuda.txt

```diff
@@ -7,5 +7,5 @@ nvidia-ml-py # for pynvml package
 torch == 2.4.0
 # These must be updated alongside torch
 torchvision == 0.19   # Required for phi3v processor. See https://github.com/pytorch/vision?tab=readme-ov-file#installation for corresponding version
-xformers == 0.0.27.post2  # Requires PyTorch 2.4.0
-vllm-flash-attn == 2.6.1  # Requires PyTorch 2.4.0
+xformers == 0.0.27.post2; platform_system == 'Linux' and platform_machine == 'x86_64'  # Requires PyTorch 2.4.0
```

Comment:
This change is because we take the CUDA requirements for the "empty" device wheel, and `xformers` and `vllm-flash-attn` are available only on Linux

Reply:
`platform_system == 'Linux'` makes sense to me.

is `platform_machine == 'x86_64'` necessary?

Reply:
`vllm-flash-attn` have published wheels only for `x86_64`, and no published tar.gz - https://pypi.org/project/vllm-flash-attn/#files
`xformers` also has wheels only for 64bit machines. It does have a tar.gz but from what I found online it can't be installed on 32bit - https://pypi.org/project/xformers/#files

So I'm pretty sure it's needed for both